src/Datadog.Trace.ClrProfiler.Managed/Integrations/HttpMessageHandlerIntegration.cs

```diff
@@ -21,54 +22,93 @@ public static class HttpMessageHandlerIntegration
         /// <param name="handler">The <see cref="HttpMessageHandler"/> instance to instrument.</param>
         /// <param name="request">The <see cref="HttpRequestMessage"/> that represents the current HTTP request.</param>
         /// <param name="cancellationTokenSource">The <see cref="CancellationTokenSource"/> that can be used to cancel this <c>async</c> operation.</param>
+        /// <param name="opCode">The OpCode used in the original method call.</param>
         /// <returns>Returns the value returned by the inner method call.</returns>
         [InterceptMethod(
             TargetAssembly = "System.Net.Http",
             TargetType = "System.Net.Http.HttpMessageHandler",
+            TargetMethod = "SendAsync",
             TargetMinimumVersion = Major4,
             TargetMaximumVersion = Major4)]
+        public static object HttpMessageHandler_SendAsync(
+            object handler,
+            object request,
+            object cancellationTokenSource,
+            int opCode)
+        {
+            // original signature:
+            // Task<HttpResponseMessage> HttpMessageHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+            var tokenSource = cancellationTokenSource as CancellationTokenSource;
+            var cancellationToken = tokenSource?.Token ?? CancellationToken.None;
+            var callOpCode = (OpCodeValue)opCode;
+            var handlerType = typeof(HttpMessageHandler); // Note the HttpMessageHandler to match the method call we replaced
```

Comment:
Same comment here, can we do `httpMessageHandlerType` instead of just `handlerType` for the variable name?

Reply:
Renamed in the same commit as the previous comment.